### PR TITLE
Adding search bar that works by searching through tags

### PR DESCRIPTION
**Pull request by QuantamHD**

### CODE REVIEW

File: src/lib/sidebar/NoteList.js [hunk 1]
```diff
@@ -18,25 +14,35 @@ class NoteList extends React.Component {
...
 }
 
 NoteList.propTypes = {};
 
 const mapStateToProps = state => {
-  return { allNotes: allNotes(state) };
+  return { 
+    allNotes: allNotes(state),
+    query: query(state)
+  };
```
Comment (kmccrohan) [inline src/lib/sidebar/NoteList.js]: I like how you made query a redux state variable.

File: src/lib/sidebar/index.js [hunk 2]
```diff
@@ -1,21 +1,49 @@
...
     );
   }
 }
 
 Sidebar.propTypes = {};
 
-export default mutate(Sidebar, "Sidebar");
+const mapStateToProps = state => {
+  return {
+    query: query(state)
```
Comment (kmccrohan) [inline src/lib/sidebar/index.js]: I don't think this query prop is used anywhere in this class...

**Comment by QuantamHD** [comment]
Fixed it.

**Comment by Flaque** [comment]
Sorry it took me so long to look at this 😅

LGTM! :D

**Pull request merged**